ownership/test_views.py

```diff
@@ -207,6 +210,67 @@ def test_get_not_allowed(self):
         response = self.client.get(url)
         self.assertEqual(response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED)
 
+    @patch('switchboard.dds_util.DDSUtil')
+    @patch('switchboard.dds_util.DDSProjectTransfer')
+    @patch('d4s2_api.utils.Message')
+    def test_receiving_user_can_accept_deliveries_without_email_template_set(self, mock_message,
+                                                                             mock_dds_project_transfer, mock_dds_util):
+        mock_message.return_value.email_text = ''
+
+        # accepting/current user should have not email template setup
+        self.user_email_template_set.delete()
+
+        # a template set is specified when the delivery was created
+        sending_user = django_user.objects.create_user('sender', password='senderpass')
+        sender_email_template_set = EmailTemplateSet.objects.create(name='group1')
+        EmailTemplate.objects.create(template_set=sender_email_template_set,
+                                     owner=sending_user,
+                                     template_type=EmailTemplateType.objects.get(name='accepted'),
+                                     subject='Subject',
+                                     body='email body')
+        delivery = DDSDelivery.objects.create(
+            project_id='project1', from_user_id='fromuser1',
+            to_user_id='touser1', transfer_id='abc123',
+            email_template_set=sender_email_template_set)
+        transfer_id = delivery.transfer_id
+
+        url = reverse('ownership-process')
+        response = self.client.post(url, {'transfer_id': transfer_id}, follow=True)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        delivery.refresh_from_db()
+        self.assertEqual(delivery.state, State.ACCEPTED)
+
+    @patch('switchboard.dds_util.DDSUtil')
+    @patch('switchboard.dds_util.DDSProjectTransfer')
+    @patch('d4s2_api.utils.Message')
+    def test_delivery_emails_always_use_senders_email_template_set(self, mock_message, mock_dds_project_transfer,
```

Comment:
High level root issue test `test_delivery_emails_always_use_senders_email_template_set `.

Reply:
The case I had in mind was to show when a sender creates a delivery via the API, their template set is used for all emails related to the delivery, regardless of other template sets in the database or if the recipient user has a template set or not. Really more of an end-to-end test for a scenario.

I see that the API lookup logic is unit tested, but thought this specific issue was a good place to have a higher level test.